src/pybind/mgr/cephadm/migrations.py

```diff
@@ -47,7 +47,8 @@ def set(self, val: int) -> None:
         self.mgr.migration_current = val
 
     def is_migration_ongoing(self) -> bool:
-        return self.mgr.migration_current != LAST_MIGRATION
+        mc: Optional[int] = self.mgr.migration_current
+        return mc is None or mc < LAST_MIGRATION
```

Comment:
it's just an artifact of it being a config option (mgr/cephadm/migration_current). We (cephadm) wouldn't set it to None but technically a user could do so so we have to account for it. That being said, None is sort of undefined behavior here currently. I think the way I changed this was to retain old behavior (None is never equal to LAST_MIGRATION so it would return True). We could take the time to define what we want that to mean/how it should be handled now that we're here though.